RadarSDK/Radar.m

```diff
@@ -44,6 +46,13 @@ + (id)sharedInstance {
 }
 
 + (void)initializeWithPublishableKey:(NSString *)publishableKey {
+    RadarInitializeOptions *options = [[RadarInitializeOptions alloc] init];
+    [Radar initializeWithPublishableKey:publishableKey options:options];
+}
+
+
++ (void)initializeWithPublishableKey:(NSString *)publishableKey
+                             options:(RadarInitializeOptions *)options {
```

Comment:
I think it's a much nicer interface to just accept a dictionary here instead of `RadarInitializeOptions`

Reply:
could also add a initialize function that takes a function to modify RadarInitializeOptions like how sentry initialization works https://github.com/getsentry/sentry-cocoa/blob/main/Sources/Sentry/SentrySDK.m.

Reply:
yes indeed — we'll make a call on this after the Thursday geofencing review